lang/test/org/partiql/lang/eval/evaluatortestframework/EvaluatorErrorTestCase.kt

```diff
@@ -55,6 +55,17 @@ data class EvaluatorErrorTestCase(
      */
     override val implicitPermissiveModeTest: Boolean = true,
 
+    /**
+     * This will be executed to perform additional exceptions on the resulting exception.
+     */
+    val additionalExceptionAssertBlock: (SqlException) -> Unit = { },
+
+    /**
+     * Determines which pipeline this test should run against; the [CompilerPipeline],
+     * [org.partiql.lang.planner.PlannerPipeline] or both.
+     */
+    override val target: EvaluatorTestTarget = EvaluatorTestTarget.ALL_PIPELINES,
```

Comment:
nit: I think `target` could be renamed to `targetPipeline` to be more specific.

Reply:
done.